blockchain/validate.go

```diff
@@ -1131,19 +1131,12 @@ func (b *BlockChain) checkBlockContext(block *exccutil.Block, prevNode *blockNod
 		// Switch to using the past median time of the block prior to
 		// the block being checked for all checks related to lock times
 		// once the stake vote for the agenda is active.
-		blockTime := header.Timestamp
-		lnFeaturesActive, err := b.isLNFeaturesAgendaActive(prevNode)
+		medianTime, err := b.index.CalcPastMedianTime(prevNode)
```

Comment:
Can we just assign it to `blockTime`? `medianTime` is not used anymore.

Reply:
Sure, good point :+1: